### PR TITLE
created header file and updated stats

### PR DESCRIPTION
**Pull request by cookandy**
added logic to determine which page is active in the nav bar
added TOTAL PLAYS to the stats

### CODE REVIEW

File: datafactory/get-library-stats.php [hunk 1]
```diff
@@ -46,5 +46,15 @@
 			echo '</li>';
 		}
 	}
+
+	$playsquery = "SELECT count(*) FROM processed";
+	$playsresults = getResults($database, $playsquery);
+	$plays = $playsresults->fetchColumn();
+
+	echo '<li>';
+		echo '<h1>'.$plays.'</h1>';
```
Comment (Arcanemagus) [inline datafactory/get-library-stats.php]: You should have spaces around the `.` operator.

File: header.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
+
+		<div class="container">
+			<div class="navbar navbar-fixed-top">
+				<div class="navbar-inner">
+					<a href="index.php"><div class="logo hidden-phone"></div></a>
+				<ul class="nav">
```
Comment (Arcanemagus) [inline header.php]: This element (and its children) should be one more indent level in.

**Comment by Arcanemagus** [comment]
Looks good other than some style nitpicks, I'll test it and merge it tonight.

**Comment by cookandy** [comment]
updated to accommodate styling nitpicks  :)

**Comment by Arcanemagus** [comment]
This looks like it should be fine, this project needs another round of cleanup along with some linter configs committed to standardize how people are committing code.

I'm merging this for now though as it doesn't look like it will break anything :+1:

**Pull request merged**